src/cli.ts

```diff
@@ -61,6 +67,22 @@ if (configResults) {
   Object.assign(options, configResults.config)
 }
 
+if (cli.flags.exportConfig) {
+  if (options.typescript) {
+    const config = getTSLintConfig(options)
+    writeFileSync('./tslint.json', JSON.stringify(config, null, 2))
+    console.log(chalk.bold.green('\n\u2714 tslint.json generated\n'))
+  } else {
+    const config = getESLintConfig(options)
+    const ignore = getESLintIgnores(options.ignore)
+    writeFileSync('./eslintrc.json', JSON.stringify(config, null, 2))
+    writeFileSync('./.eslintignore', JSON.stringify(ignore.join('\n'), null, 2))
+    console.log(chalk.bold.green('\n\u2714 eslintrc.json generated\n'))
+    console.log(chalk.bold.green('\u2714 .eslintignore generated\n'))
+  }
+  process.exit(0)
+}
```

Comment:
Sorry just a few last nitpicks:

1. Instead of `cli.flags.exportConfig`, can you use `options` again like `options.exportConfig`
2. `ignore.join` returns a string, so `JSON.stringify` isn't necessary for that part.
3. Can you remove the last `\n` in `console.log(chalk.bold.green('\n\u2714 eslintrc.json generated\n'))`? This was my bad, I forgot that having a second `console.log` will already put the next log in the next line.